src/templates/index.hbs

```diff
@@ -118,9 +118,9 @@
         </div>
         <div class="row">
           <div class="col-md-4">
-                <img src="http://slack-opensanca.herokuapp.com/badge.svg">
+                 
                   <a href="https://slack-opensanca.herokuapp.com/"><i class="fa fa-slack fa-5x"></i></a>
-                  <h3>Slack</h3>
+                  <h3>Slack <img src="http://slack-opensanca.herokuapp.com/badge.svg"></h3>
```

Comment:
All absolute resource URIs should use the HTTPS protocol, otherwise browsers will complain and warn users that their browsing is not secure (as is happening right now at least if you use Chrome). As this is not directly related to this PR I'm going to create another issue to tackle this.

Reply:
Done: #27.